android/src/main/java/io/customer/reactnative/sdk/CustomerIOReactNativeInstance.kt

```diff
@@ -93,5 +98,11 @@ object CustomerIOReactNativeInstance {
         )
     }
 
-    private fun configureModuleMessagingInApp() = ModuleMessagingInApp()
+    private fun configureModuleMessagingInApp(
+        inAppEventListener: InAppEventListener? = null,
```

Comment:
i think it can't be nullable considering [this](https://github.com/customerio/customerio-reactnative/pull/89/files#r1099737341)